oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -404,6 +398,50 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    private Map<String, String> buildDimensions(
```

Comment:
This is similar to something done as part of the [ATO-133/ATO-134](https://github.com/govuk-one-login/authentication-api/pull/3658) - it is in the shared location.
Happy to make adjustments to better align with this so that only a shared method is needed :)

Reply:
https://github.com/govuk-one-login/authentication-api/pull/3658/files#:~:text=Map%3CString%2C%20String%3E%20getDimensions

Reply:
I think this one will go in first. If you get a chance to refactor this to use your shared method that would be great, if not no big deal 🙂 Thanks!